crates/ty_static/src/env_vars.rs

```diff
@@ -39,6 +40,12 @@ impl EnvVars {
     /// when necessary, e.g. to watch for file system changes or a dedicated UI thread.
     pub const TY_MAX_PARALLELISM: &'static str = "TY_MAX_PARALLELISM";
 
+    /// Used to enable persistent caching at the specified path.
+    ///
+    /// This improves performance and memory usage by caching to disk across ty runs.
```

Comment:
Let's add a note that this is highly experimental and currently makes ty slower.